src/site/constants/buckets.js

```diff
@@ -1,7 +1,7 @@
 const { VAGOVDEV, VAGOVSTAGING, VAGOVPROD } = require('./environments');
 
 module.exports = {
-  [VAGOVDEV]: 'https://dev-va-gov-assets.s3-us-gov-west-1.amazonaws.com',
+  [VAGOVDEV]: 'https://s3-us-gov-west-1.amazonaws.com/apps.dev.va.gov',
```

Comment:
```suggestion
  [VAGOVDEV]: 'https://apps.dev.va.gov.s3-us-gov-west-1.amazonaws.com',
```

Reply:
@U-DON that format doesn't actually work: https://apps.dev.va.gov.s3-us-gov-west-1.amazonaws.com/generated/file-manifest.json

Reply:
I'm able to access it, but it does have a cert issue. Didn't realize putting the bucket name in the path was a thing.

Reply:
Yeah it's kind of a weird thing, but this is the discussion with ops https://dsva.slack.com/archives/CBU0KDSB1/p1615926861056600?thread_ts=1615827961.141400&cid=CBU0KDSB1